lib/plugins/features/movies/controller.js

```diff
@@ -7,3 +7,34 @@ exports.create = async (payload) => {
 
   return new Movie({ id: movie.id }).fetch();
 };
+
+exports.retrieve = async (queryParams) => {
+  return new Movie().query((qb) => {
+
+    if (queryParams) {
+      if (queryParams.year) {
+        // Query by a particular year
+        qb.where('release_year', '=', queryParams.year);
+      }
+      if (!queryParams.year) {
```

Comment:
We could maybe use an `else` here? since these two conditions seem to be mutually exclusive?

Reply:
You're right.